src/onnx/onnx.cpp

```diff
@@ -100,11 +100,17 @@ struct onnx_parser
 
     void init_actv_func()
     {
+        // Support name format of all lower case or the first letter capital
```

Comment:
Is this necessary for the ops we have currently?

Reply:
Yes, the activate function is used in the rewrite_rnn function, we cannot just all the various parse_XXX function to process them...